### DISCUSSION TITLE
kafka-ui helm chart does not base64 encode envs.secret

### DISCUSSION THREAD

**Discussion by cignul9** [provectus/kafka-ui#2389] [General]
When using envs.secret to store...secrets the chart fails because values are not encoded in the template.  We can encode them beforehand, but that's not very user friendly.  Would you consider changing kafka-ui/charts/kafka-ui/templates/secret.yaml line 9 from 
```
  {{- toYaml .Values.envs.secret | nindent 2 }}
```
to
```
  {{- toYaml .Values.envs.secret | b64enc | nindent 2 }}
```
?

**Comment by Haarolean** [comment]
Hey, thanks for reaching out. I'll convert this to an issue.
